### PR TITLE
Issue/297sync

### PR DESCRIPTION
**Pull request by LauraSteiner**
@jwahlin Ready for next round  of reviews.  My responses to your many excellent comments are in the issue discussion:

https://github.com/Strategy11/formidable/issues/297

Thank you! 

(No changes from earlier.  I'm just turning this into a pull request for easier review.)

### CODE REVIEW

**Comment by LauraSteiner** [comment]
@jwahlin Ready for your next round of feedback!

File: js/formidable_admin.js [hunk 1]
```diff
@@ -2978,6 +2979,17 @@ if(f){
 }
 }
 
+function checkRepeatLimit() {
```
Comment (stephywells) [inline js/formidable_admin.js]: Would you mind moving this function inside the closure? It would be great right after the toggleRepeatButtons function.

**Comment by LauraSteiner** [comment]
Thank you for the review, Steph!  I believe I've made the change you requested.  

I'm ready for further review from you and Jamie!

File: css/frm_admin.css [hunk 2]
```diff
@@ -1677,6 +1677,10 @@ span.howto{display:inline;}
 	display:none;
 }
 
+.frm_add_form_row.frm_hide_add_button {
+	display: none;
+}
+
```
Comment (jwahlin) [inline css/frm_admin.css]: Does this have a purpose? If so, when/where does it apply?

File: js/formidable_admin.js [hunk 3]
```diff
@@ -988,6 +988,17 @@ function frmAdminBuildJS(){
 		}
 	}
 
+	function checkRepeatLimit() {
+
+		if (this.value >= 2 && repeatLimit <= 200) {
```
Comment (jwahlin) [inline js/formidable_admin.js]: The repeatLimit variable isn't defined. This should be this.value.

File: js/formidable_admin.js [hunk 4]
```diff
@@ -988,6 +988,17 @@ function frmAdminBuildJS(){
...
+			return;
+		}
+
+		alert("Please enter an integer greater than 2 for the Repeat Limit.");
```
Comment (jwahlin) [inline js/formidable_admin.js]: This should say "greater than 1", right?

File: css/frm_admin.css (hunk 2 above)
Comment (LauraSteiner) [reply css/frm_admin.css]: Not sure what this is doing here.  I can't see a purpose.  I'm removing it.

File: js/formidable_admin.js (hunk 3 above)
Comment (LauraSteiner) [reply js/formidable_admin.js]: Thanks!  I missed that when I rewrote.

File: js/formidable_admin.js (hunk 4 above)
Comment (LauraSteiner) [reply js/formidable_admin.js]: Yes!   Another great catch!

 I'm concerned that some people may not know what the word "integer" means, so maybe I'll use "greater than or equal to 2."  Then they'll see the number 2 and hopefully get that that's the minimum.

**Pull request merged**